precode.sh

```diff
@@ -7,37 +7,76 @@
 #   dir3
 #       dir4
 
+mkdir task
```

Comment:
Можно и с переходами, но проще одной командой

```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```